Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -1250,11 +1250,9 @@ static bool isInRecoveryOS()
 #if HAVE(SUPPORT_HDR_DISPLAY_APIS)
 static void setDynamicRangeLimitRecursive(CALayer* layer, LayerDynamicRangeLimitSetter layerDynamicRangeLimitSetter)
 {
-    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    if ([layer wantsExtendedDynamicRangeContent]) {
-    ALLOW_DEPRECATED_DECLARATIONS_END
+    if ([layer toneMapMode] == CAToneMapModeIfSupported)
         layerDynamicRangeLimitSetter(layer);
```

Comment:
this is where we set CADynamicRangeStandard in the UI process @smfr

Reply:
So we need to keep `WebViewImpl::applicationShouldSuppressHDR()` (above in this file) and `GraphicsLayerCA::hdrContentsFormat()` to stay absolutely in sync, so that they always do the same thing when suppressing HDR or not!
Would it be possible to keep this logic in one common place? (Maybe in a follow-up bug, in which case some scary comments would be nice here, so we don't modify one and not the other by mistake.)

Reply:
I ended up changing the approach so no need for the comment I think?